# Request message
geometry_msgs/PointStamped[] points

# Response message
---

bool status
